Tests/OpenCombineTests/PublisherTests/DecodeTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  DecodeTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 6/21/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class DecodeTests: XCTestCase {
```

Comment:
As I've already mentioned, `Decode` and `Map` have almost the same semantics, which means you can steal [some tests for `Map`/`TryMap`](https://github.com/broadwaylamb/OpenCombine/blob/e3146a1efba678e6a3716ac7dc21958e7def2a78/Tests/OpenCombineTests/PublisherTests/MapTests.swift).